ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/button/v1/button/button.html

```diff
@@ -34,9 +34,9 @@
       disabled="${!button.enabled}"
       title="${button.tooltipVisible ? '' :   button.tooltipText}"
       data-cmp-data-layer="${button.data.json}"
-      aria-label="${button.label.value}">
+      aria-label="${button.label.visible != false ? '' : button.label.value}">
```

Comment:
Please add test case for this change, we already saw test failures because of this.

Add test case for both scenarios.
1. when cq dialog is not openend.
2. when cq dialog is openend and saved